src/components/BookList.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+function BookList({ books, onDelete }) {
+  return (
+    <div>
+      <h1>Book List</h1>
+      <ul>
+        {books.map((book) => (
+          <li key={book.id}>
+            <Book book={book} onDelete={onDelete} />
+          </li>
+        ))}
+      </ul>
+    </div>
+  );
+}
+
```

Comment:
- [ ] This component is supposed to Renders a list of books using the `<Book>` component